src/test/resources/rca/rca_muted.conf

```diff
@@ -44,5 +44,5 @@
     "rotation-period-seconds": 21600
   },
 
-  "muted-rcas": "CPU_Utilization, Heap_AllocRate"
+  "muted-rcas": ["CPU_Utilization, Heap_AllocRate"]
```

Comment:
Do we not want this instead -
`"muted-rcas": ["CPU_Utilization", "Heap_AllocRate"]` ?

Reply:
Correct, fixing this.